measure.rb

```diff
@@ -266,6 +266,8 @@ def self.create(hpxml_doc, runner, model, weather, map_tsv_dir)
     @hvac_map = {} # mapping between HPXML HVAC systems and model objects
     @dhw_map = {}  # mapping between HPXML Water Heating systems and model objects
 
+    @related_hvac_list = [] # list of hvac systems refered in water heating system "RelatedHvac" element
```

Comment:
Since this variable is only used in the `add_hot_water_and_appliances`, can we define it there instead of making it a global?